content/en/docs/components/pipelines/v2/concepts/output-artifact.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Output Artifact"
```

Comment:
I feel like Artifact itself is a concept, while input vs. output doesn't add anything special on top of the artifact concept.